content/docs/en/elements/components/side-drawer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SideDrawer
+apiRef: https://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/SideDrawer/overview
+contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska, sombriks]
+---
+
+`<RadSideDrawer>` is a UI side drawer component which plays a navigation role on application structure.
+
+## Installing the side drawer:
+
+```shell
+$ tns plugin add nativescript-ui-sidedrawer
+```
+
+After that, you need to register it as a vue component:
```

Comment:
Specify where this code needs to be added. Is it at the top of some file or code block in a file, which file?

No colon at the end of the sentence.